tests/Commands/__snapshots__/ComponentClassCommandTest__it_generated_correct_file_with_content__1.php

```diff
@@ -0,0 +1,26 @@
+<?php return '<?php
+namespace Modules\\Blog\\View\\Component;
+use Illuminate\\View\\Component;
+class Blog extends Component
+{
+    /**
+     * Create a new component instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
```

Comment:
Remove unused function

Reply:
this is the default component class created by laravel

Reply:
Oh... didn't know